python/nx-cugraph/nx_cugraph/convert.py

```diff
@@ -74,7 +100,8 @@ def from_networkx(
     as_directed: bool = False,
     name: str | None = None,
     graph_name: str | None = None,
-) -> nxcg.Graph:
+    use_compat_graph: bool | None = False,
```

Comment:
curious why the default for this is False

Reply:
If you already have a networkx graph, then what's the point of creating a "compat" graph from it? The user would probably be better off simply using the networkx graph with caching. Using False also minimizes the number of changes we need to make, since we use this a lot internally.